config-example.yaml

```diff
@@ -80,6 +80,13 @@ derp:
     # The Headscale server_url defined above MUST be using https, DERP requires TLS to be in place
     enabled: false
 
+    # If true, derp will reject other unknown tailscale client connections
+    verify_clients: true
+
+    # If true, headscale will simulate a socket API, which allows client authentication to be
+    # completed without running tailscaled independently.
+    enable_socket_api: false
```

Comment:
If you managed to implement the alternative socket route as proposed, I'm tempted to say that we don't have this option, and always use the simulated socket if someone turns on verify_clients.

If not, I think this needs a better name.

Reply:
It is currently unlikely to adjust the fake socket file path 😥. Do you have any suggestions for naming this field? (I apologize for English not being my native language.)